src/app.js

```diff
@@ -306,12 +445,19 @@ fluid.defaults("gpii.app.tray", {
 /**
   * Creates the Electron Tray
   * @param icon {String} Path to the icon that represents the GPII in the task tray.
+  * @param handler {Function} A handler to be used for tray menu left click.
   */
-gpii.app.makeTray = function (icon) {
+gpii.app.makeTray = function (icon, handler) {
```

Comment:
I think this function should either be renamed - it's doing more than making the tray now - or split into two functions. If you split it, then it might be better to fire an event on click that the handler can be registered to in configuration.

Reply:
Done. The function was renamed and split into two functions - one for loading the window's content and one only for showing the window.